File: C++/Actor.cpp
#include <cstring>
#include <iostream>
#include "Actor.h"

using namespace std;

Actor::Actor() {

	strcpy(this->nombre, "");
	this->nacimiento = 0;
};

Actor::Actor(char nombre[], int nacimiento) {
	strcpy(this->nombre, nombre);
	this->nacimiento = nacimiento;
};

char* Actor::getNombre() {
	return this->nombre;
}

void Actor::setNombre(char nombre[]){
	strcpy(this->nombre, nombre);
};

int Actor::getNacimiento() {
	return this->nacimiento;
}

void Actor::setNacimiento(int nacimiento) {
	this->nacimiento = nacimiento;
}

void Actor::mostrarActor() {
	cout<<"El nombre del Actor es "<<this->getNombre()<<endl;
	cout<<"Su anyo de nacimiento es "<<this->getNacimiento()<<endl;
	cout<<endl;
}

File: JAVA/peliculas/src/main/java/Actor.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Ricardo Mendez
 */
public class Actor {
    private String nombre;
    private int nacimiento;
    String mostrarActor;
    
    public Actor() {
        this.nombre = "";
        this.nacimiento = 0;
    }
    
    public Actor(String nombre, int nacimiento) {
        this.nombre = nombre;
        this.nacimiento = nacimiento;
    }
    
    public String getNombre() {
        return this.nombre;
    }
    
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    public int getNacimiento() {
        return this.nacimiento;
    }
    
    public void setNacimiento(int nacimiento) {
        this.nacimiento = nacimiento;
    }
    
    public void mostrarActor() {
        System.out.println("El nombre del autor es " + this.getNombre());
        System.out.println("Su anyo de nacimiento es " + this.getNacimiento());
    }
}
